cincinnati/src/plugins/registry.rs

```diff
@@ -0,0 +1,35 @@
+//! Plugin registry.
```

Comment:
Self note: check if there is a better term instead of "registry" (which could be confusing due to docker-registry).